udata/core/spatial/models.py

```diff
@@ -64,6 +87,17 @@ def keys_values(self):
                 keys_values.append(value)
         return keys_values
 
+    @property
+    def url(self):
+        return url_for('gouvfr.territory', territory=self)
```

Comment:
Can't we think of something like `return url_for('%s.territory' % theme, territory=self)` where theme would come from the config for example?